.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/FineTuningTests.cs

```diff
@@ -332,29 +329,17 @@ private async Task<OpenAIFileInfo> UploadAndWaitForCompleteOrFail(FileClient fil
         return uploadedFile;
     }
 
-    private Task<FineTuningJob> WaitForJobToEnd(FineTuningClient client, FineTuningJob job)
```

Comment:
Did this test become redundant given new APIs?

Reply:
exactly. This was just a helper function that someone wrote to wait for the end of the operation

Reply:
Cool.  I love it when we are able to rework the APIs to reduce the code a user has to write!